tests/e2e/features/transactions/deposit/bridge-deposit-with-blockchain.feature

```diff
@@ -0,0 +1,18 @@
+@deposit @regression @transactions @blockchain
+Feature: Deposit
+
+  Background:
+    Given Connect Metamask extension with login action
+
+  @id785
```

Comment:
I suppose we need to align steps here and in Allure https://allure.matterlabs.dev/project/17/test-cases/785?treeId=30 (allure looks poor)